src/ripples/circle-ripple.jsx

```diff
@@ -25,11 +35,29 @@ const CircleRipple = React.createClass({
 
   getDefaultProps() {
     return {
-      color: Colors.darkBlack,
       opacity: 0.16,
     };
   },
 
+  getInitialState() {
```

Comment:
I'll take a look at what's involved, first pass I was just trying to remove all hard-coded values.